src/geo/lng_lat.js

```diff
@@ -130,8 +130,7 @@ class LngLat {
     toEcef(altitude: number): [number, number, number] {
         const altInEcef = globeMetersToEcef(altitude);
         const radius = GLOBE_RADIUS + altInEcef;
-        const ecef = latLngToECEF(this.lat, this.lng, radius);
-        return [ecef[0], ecef[1], ecef[2]];
+        return latLngToECEF(this.lat, this.lng, radius);
```

Comment:
Apparently this was to silence the Flow error (`Vec3` vs the more strict `[number, number, number]` incopatibility) — let's try to fix the typing rather than add an actual new allocation.

Reply:
I was getting flow errors in VS code so I did this as a workaround. Fixing the type is the correct way to go ofc.